marketplace-client/src/marketplace/src/app/_models/staff-member.ts

```diff
@@ -0,0 +1,12 @@
+import {Role} from './role';
+
+export interface StaffMember {
+  name?: string;
+  surname?: string;
+  password?: string;
```

Comment:
do we store passwords just as is? is it secure?

Reply:
Thanks for noticing that. Password shouldn't be there in the first place. We don't need it for the process of registering staff member (password generation link is sent directly to the staff member)